index.html

```diff
@@ -20,11 +20,19 @@
                             <li><a href="#">Portfolio</a></li>
                             <li><a href="#">About</a></li>
                             <li><a href="#">Contact</a></li>
-                            <i class="fa-solid fa-bars"></i>
                         </ul>
+                    <div class="container accordion" id="accordion" onclick="myFunction(this)">
+                        <div class="bar1"></div>
+                        <div class="bar2"></div>
+                        <div class="bar3"></div>
                     </div>
             </nav>
         </header>
+        <div id="panel">
+            <a href="#recent-work" class="drop-link">Portfolio</a>
+            <a href="#about" class="drop-link">About</a>
+            <a href="#contact" class="drop-link">Contact</a>
+        </div>
```

Comment:
- [ ] As pointed out by the previous reviewer,  your mobile menu isn't identical to the mobile menu of the Figma template you chose. Please, make the mobile menu the full view height of the mobile screen to cover the whole mobile screen. 👍 💻. This gives a more appealing user view and gives a better experience.

## View:
![image](https://user-images.githubusercontent.com/50990925/223311008-02fd7497-fed6-4e55-842f-b5a31b58b801.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/223311055-e36f4c6c-a091-4da5-ae37-c97f874da389.png)

Reply:
Applied this as per reuqirement